class2.html

```diff
@@ -432,15 +440,20 @@ <h6 class="grey">The Homework was:</h6>
                     <p>For example:</p>
                     <pre><code  class="ruby">
                       if x > 10
-                      puts "x is greater than 10"
-                      if x < 20
-                      puts "but x is less than 20"
-                      end
-                      elsif x <= 10 && x > 0
-                      puts "x is a number between 1 and 10"
+                        puts "x is greater than 10"
+                      elsif x < 20
+                        puts "but x is less than 20"
+                      elsif 0 < x && x <= 10
+                        puts "x is a number between 1 and 10"
                       else
-                      puts "Wow, don't be so negative, dude"
+                        puts "Wow, don't be so negative, dude"
                       end
+
+                      # Note!
+                      # You must use separate expressions:
+                      # You might be tempted, instead of  0 < x && x <= 10
+                      # to write 0 < x <= 10 (like you might in math)
+                      # This will cause errors.
```

Comment:
So, this actually a really good thing to highlight - I wonder if it bears existing on it's own slide?

We could call out why it doesn't work (the first comparison returns a bool, the next one fails because bool can't be < a number, etc), with an example of what not to do, then what to do (like you have here).

Just a thought, not 100% required. We can talk through it as well, or have people try it on their own and "figure out" what went wrong.